src/Entities/Invoice.php

```diff
@@ -73,6 +73,24 @@ public function setWebsiteId($value)
         return $this->setAttribute('websiteId', $value);
     }
 
+    /**
+     * @return string
+     */
+    public function getSubscriptionId()
+    {
+        return $this->getAttribute('subscriptionId');
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function setSubscriptionId($value)
```

Comment:
I maybe wrong, but in specs the `subscriptionId` is readonly property?

Reply:
Fixed in https://github.com/Rebilly/rebilly-php/pull/323